bool deliverySwitch
bool hopperSwitch
bool doserSwitch
int8 doserSpeed
bool wettingSwitch
bool mixerSwitch
int8 mixerSpeed
bool pumpSwitch
int8 pumpRotarySpeed
int8 pumpUpDownSpeed

bool eStop
bool tower2Cart
bool cart2Tower
int8 RoboClaw5PWM
int8 RoboClaw6PWM
